crates/cdk/src/nuts/nut00/mod.rs

```diff
@@ -389,6 +397,10 @@ impl CurrencyUnit {
             Self::Msat => Some(1),
             Self::Usd => Some(2),
             Self::Eur => Some(3),
+            Self::Ngn => Some(4),
+            Self::Ghs => Some(5),
+            Self::Kes => Some(6),
+            Self::Zar => Some(7),
```

Comment:
Thanks for this PR. This isn't an issue introduced by this PR but the current way we do derivation path by just assigning numbers does not scale well and I don't think we should add more with this method. We've discussed the idea of using a short hashing method to achieve this. https://github.com/cashubtc/cdk/pull/464#pullrequestreview-2449511528